nest/src/users/users.controller.ts

```diff
@@ -40,7 +40,7 @@ import {
 } from './utils/upload-utils';
 
 @Controller('users')
-@UseGuards(JwtGuard)
+// @UseGuards(JwtGuard)
```

Comment:
why did you remove the guard from here ?

Reply:
yeah that's probably to run some requests on swagger when the user isn't authenticated yet